docs/api/swagger.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "vocascan",
     "description": "A highly configurable vocabulary trainer.",
-    "version": "1.0.0"
+    "version": "v0.1-alpha"
```

Comment:
?

Reply:
I pasted some code, which i removed, but seems like I accidentally overlooked that I changed the version

Reply:
done